stories/CCard.stories.ts

```diff
@@ -184,3 +205,43 @@ export const Color: Story = {
     slotTemplate: "Cards main content.",
   }),
 };
+
+export const ScrollableContent: Story = {
```

Comment:
1. How can I set the padding? Which padding is it?
2. Is the padding before the first and after the last list item intended? IMO it looks kind of odd

Reply:
1. I am not sure, what you mean. You get the padding (padding left and right, which the CCard sets by default for every content except the "full-width-content") by setting it like `:class="padding"`. This is used for when you want to add the defaults card padding to some content inside the full-width content. Example of a list: the dividers should be full width, but the actual content inside the items should have the padding.
2. i don't understand that question, the padding is set to every item the same isn't it?

Reply:
1. Got it, thanks
2. Take a look
![image](https://github.com/Crystal-Creations-GbR/crystal-components/assets/79226886/d2e7e4d6-7780-4e7e-ae3e-97108d0d1fe0)
![image](https://github.com/Crystal-Creations-GbR/crystal-components/assets/79226886/0df0265b-9266-4fa8-8e20-3a6838659ab4)
Is this intended?

Reply:
yes, thanks for that, that does not look good, I made it variable so that it looks good for the most content by default and we can change it for special components like the list